bpf/bpf_host.c

```diff
@@ -1308,8 +1308,10 @@ int cil_from_netdev(struct __ctx_buff *ctx)
 		return CTX_ACT_OK;
 #endif
 	ret = tcx_early_hook(ctx, proto);
-	if (ret != CTX_ACT_OK)
-		return ret;
+	if (ret != CTX_ACT_OK) {
+		ret = DROP_HOOK;
+		goto drop_err;
+	}
```

Comment:
```suggestion
	if (IS_ERR(ret))
		goto drop_err;
```

Let's just pass through whatever error we receive. `DROP_HOOK` wouldn't help much in debugability.

Reply:
Ok, currently the way this is used, it only passes down CTX_ACT_ but not errors, so this needs a rework first.